### PR TITLE
Add generic rug messages

### PR DESCRIPTION
**Pull request by tanya-coding**
Related to #2

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -5,9 +5,18 @@
...
+            true,
+            "never-prefix"
+        ],
         "max-classes-per-file": false,
-        "object-literal-sort-keys": false
+        "object-literal-sort-keys": false,
+        "ordered-imports": [
+            true,
+            {
+                "import-sources-order": "any"
```
Comment (ddgenome) [inline tslint.json]: If you don't like this rule, you can separate the imports by an empty line to allow a specific order. In other words, this rule only requires order within "groups" of imports.

File: tslint.json [hunk 2]
```diff
@@ -5,9 +5,18 @@
...
+            }
+        ]
     },
     "rulesDirectory": []
-}
+}
```
Comment (ddgenome) [inline tslint.json]: Why no newline?

File: src/RugMessages.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        return new ResponseMessage(`Error rendering error message ${ex}`);
+    }
+}
+
+export function renderSuccess(msg: string): ResponseMessage {
+    try {
+        const slackMessage = { text: msg };
+        return new ResponseMessage(render(slackMessage), MessageMimeTypes.SLACK_JSON);
+    } catch (ex) {
+        return new ResponseMessage(`Error rendering error message ${ex}`);
```
Comment (ddgenome) [inline src/RugMessages.ts]: Ironic

File: package.json [hunk 4]
```diff
@@ -35,4 +35,4 @@
   "dependencies": {
     "@atomist/rug": "^1.0.0-m.3"
   }
-}
+}
```
Comment (ddgenome) [inline package.json]: No newline?

File: tslint.json (hunk 1 above)
Comment (tanya-coding) [inline tslint.json]: I am not able to separate those by a new line while using sort-imports plugin that was suggested as common "best practices" settings (https://atomist-community.slack.com/files/kipz/F57TXD2QJ/vs_code_user_settings.js), specifically:

```
  "typescript.extension.sortImports.quoteStyle": "double",
  "typescript.extension.sortImports.sortOnSave": true,
  "typescript.extension.sortImports.sortMethod": "path",
```

It rearranges imports in this specific order on save :(

File: tslint.json (hunk 2 above)
Comment (tanya-coding) [inline tslint.json]: Again "best practice" settings removed new line on save

File: package.json (hunk 4 above)
Comment (tanya-coding) [inline package.json]: Same here - auto-formatting gets rid of new line

File: tslint.json (hunk 2 above)
Comment (kipz) [inline tslint.json]: That's odd - doesn't for me....

**Comment by kipz** [comment]
LGTM

File: tslint.json (hunk 1 above)
Comment (ddgenome) [inline tslint.json]: So if you separate two import statements with an empty line, that plugin removes the empty line and sorts them all together?

**Pull request merged**